jenkins/ci.suse.de/openstack-mkcloud.yaml

```diff
@@ -61,7 +61,7 @@
           default:
           description: >-
             Defines which scenario file should be used. Only works
-            with cloud5 or newer.  Currently only the 'batch' step uses
+            with cloud6 or newer.  Currently only the 'batch' step uses
```

Comment:
Why changing facts? Its just documentation anyway.
Changing it would make others assume that the scenario (crowbar batch) feature does not work on cloud5 - but it does.